api_tests/config.py

```diff
@@ -22,10 +22,11 @@ def parse():
 
     # Make sure config file has all required keys
     missing_key = verify_keys(config_dict, [
-      'url',
-      'access_token',
-      'snippet',
-      'user_id',
+        'url',
+        'access_token',
+        'snippet',
+        'user_id',
+        'invalid_snippet_key',
```

Comment:
We use the `snippet_id` we get back from the first POST test, which is based on the `snippetTitle` field of the `snippet` in the `config.json`, with a possible postfix (`-1`, `-2`, etc...).